src/app/time-picker/time-picker.sample.html

```diff
@@ -51,5 +50,27 @@ <h4 class="sample-title">Templated Time Picker</h4>
                 </igx-time-picker>
             </div>
         </article>
+        <article class="timepicker-column">
```

Comment:
It makes sense to have such sample in the input or input group sample, not in the time picker. You could leave this change but please add it also in the input or input group sample.